src/metabase/query_processor/middleware/annotate.clj

```diff
@@ -471,13 +474,34 @@
    (cols-for-ags-and-breakouts inner-query)
    (cols-for-fields inner-query)))
 
-(declare mbql-cols)
+(def preserved-keys
```

Comment:
Any chance we can make this `^:private`? As far as I can tell it's not used in any other namespaces besides this one. The main reason is so this is easier to refactor in the future. If it's public and we want to do something like move/remove/rename it we have to grep the whole codebase for `preserved-keys` and then determine whether or not matching symbols refer to this name (actually I guess maybe LSP can do this now for you now as long as we're not resolving those references dynamically -- but CIDER can't do it yet)

Reply:
Done. I think at one point it was shared and never got knocked back to private.

aside, but i thought CIDER had a find usages feature now.

Reply:
If it does I haven't figured out how to use it 💀